stages/org.osbuild.bootupd

```diff
@@ -25,9 +25,10 @@ SCHEMA_2 = r"""
   "type": "array"
 },
 "options": {
-  "additionalProperties": true,
+  "additionalProperties": false,
```

Comment:
Just a git history nitpick:
Should we squash this with the commit that introduced the stage?
Same for L31 below (`"type": "object"`).

Reply:
new upload